corehq/apps/app_manager/suite_xml/post_process/instances.py

```diff
@@ -58,6 +59,12 @@ def add_referenced_instances(self, entry, details_by_id, relevance_by_menu, menu
         xpaths.discard(None)
         instances, unknown_instance_ids = EntryInstances.get_required_instances(xpaths)
 
+        if toggles.CUSTOM_CALENDAR_FIXTURE.enabled(self.app.domain):
```

Comment:
Do you think this is an argument for not hiding the calendar fixture behind a feature flag? Otherwise, I might pull this out into a helper function `_add_custom_referenced_instances`.